### PR TITLE
Lizards only triple single s instead of double ss or more.

### PR DESCRIPTION
**Pull request by KazeEspada**
Lizards only triple single s instead of double ss or more. This makes them slightly less annoying.

I mean they are dirty lizardssssssssssssssssssss and they had this coming.

### CODE REVIEW

**Comment by bgobandit** [comment]
lizards will never be less annoying

**Comment by Zilenan91** [comment]
So as an example, how would this change how lizards speak.

If they want to say sardines how would it turn out

**Comment by KazeEspada** [comment]
Assess = Assess
Sardines = Sssardinesss
Assisstants = Assisstantsss

**Comment by duncathan** [comment]
You should make groups of two turn into three as well

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -82,9 +82,9 @@ datum/species/human/spec_death(gibbed, mob/living/carbon/human/H)
 	return 1
 
 /datum/species/lizard/handle_speech(message)
-	// jesus christ why
+
 	if(copytext(message, 1, 2) != "*")
-		message = replacetext(message, "s", "sss")
+		message = regex_replaceall(message, "(?<!s)s(?!s)", "sss") //(?<!s) Not s before. (?!s) not s after. That way it only triples a single s instead of double ss.
```
Comment (Incoming5643) [inline code/modules/mob/living/carbon/human/species_types.dm]: but can your precious regex make it so upper case `S` is followed by other uppercase `S`'s

**Comment by KazeEspada** [comment]
How important would turning Ass into Asss be? Also, As and Ass would become the same word.

File: code/modules/mob/living/carbon/human/species_types.dm (hunk 1 above)
Comment (KazeEspada) [inline code/modules/mob/living/carbon/human/species_types.dm]: It would take two regex, but I could.

**Comment by Bawhoppen** [comment]
Insssssstead of making there be lessssssssssss "S"ssssss, you sssssshould make there be more.

File: code/modules/mob/living/carbon/human/species_types.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species_types.dm]: this is a case insensitive regex comparison

**Pull request merged**